README.md

```diff
@@ -16,6 +16,9 @@ details.
 
 - JupyterLab >= 3.0
 
+> [!IMPORTANT]
```

Comment:
The line above this says that "JupyterLab >= 3.0" is required. To clarify that JupyterLab 4 is supported on the main branch, should we instead say that Lab 4 is required?

Reply:
Makes sense and it would encourage people to update to JupyterLab 4.0 even more. Updated.